app/src/main/java/com/fstyle/structure_android/screen/userdetail/UserDetailPresenter.java

```diff
@@ -1,5 +1,11 @@
 package com.fstyle.structure_android.screen.userdetail;
 
+import com.fstyle.structure_android.data.model.User;
```

Comment:
ERROR: Unused import: com.fstyle.structure_android.data.model.User. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports